src/App/Pages/Accounts/LockPage.xaml.cs

```diff
@@ -64,6 +69,13 @@ public async Task PromptBiometricAfterResumeAsync()
         protected override async void OnAppearing()
         {
             base.OnAppearing();
+            _broadcasterService.Subscribe(nameof(LockPage), async (message) =>
+            {
+                if (message.Command == Constants.ClearSensitiveFields)
+                {
+                    Device.BeginInvokeOnMainThread(ResetPinPasswordFields);
+                }
+            });
```

Comment:
Either mark as `async` and use `await Device.InvokeOnMainThreadAsync(...)` or remove the `async` keyword. As it's now, it doesn't serve any purpose.
Also, if it's marked as `async` then the whole body should be wrapped in a `try...catch` to prevent crashes if an exception arises